src/xmrEngine.js

```diff
@@ -444,7 +447,7 @@ class MoneroEngine {
 
   log(...text: Array<any>) {
```

Comment:
This whole method needs to be deleted. It will never actually be called, because it gets stomped by the `this.log =  opts.log` in the constructor. That's fine, though, since the core now includes the `walletId` for ya.